website/frontend/templates/base.html

```diff
@@ -6,6 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="{{ _('Official website for MusicBrainz Picard, a cross-platform music tagger written in Python.') }}">
     <meta name="author" href="{{ url_for('humans.show_humans') }}">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/js-cookie/3.0.1/js.cookie.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.7.0.js"></script>
+    <link rel="stylesheet" type="text/css" href="https:///cdn.datatables.net/1.13.7/css/jquery.dataTables.min.css">
+    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.13.7/js/jquery.dataTables.min.js"></script>
```

Comment:
Instead of globally including the scripts in the header there is already a `scripts` block in the base.html file.

You can use this to inject the dependencies only for the plugins page. In `plugins.html` add the following:

```
{%- block scripts %}
{{ super() }}
... add the script tags here ...
{% endblock -%}
```

Reply:
Thank you, @phw, for reviewing the pull request. I appreciate your feedback and agree with your suggestion that it would be ideal to load the necessary scripts, only for that particular page, without loading it globally, and make it so that the script is not loaded via a CDN, but rather by loading the actual script itself.

I plan to address these issues as early as possible.